ziti/src/main/kotlin/org/openziti/impl/ZitiContextImpl.kt

```diff
@@ -377,7 +377,7 @@ internal class ZitiContextImpl(internal val id: Identity, enabled: Boolean) : Zi
 
     }
     private fun getDnsTarget(addr: InetSocketAddress): String? {
-        if (addr.address != null) {
+        if (addr.hostString == null && addr.address != null) {
```

Comment:
need to re-order the checks

* check `addr.hostString != null`
  *  isValidIP() -> null
  *  else -> hostString
* else do reverse lookup

Reply:
Ok.   Done.